### DISCUSSION TITLE
How to sign Cocoapods

### DISCUSSION THREAD

**Discussion by nmtx-2022** [codemagic-ci-cd/codemagic-docs#1004] [Q&A] [answered]
![图片](https://user-images.githubusercontent.com/109792150/193050033-2e484b79-fcf1-434f-9332-249582781590.png)

Does anyone know how to solve it in codemagic, thanks!

**Comment by K-Codemagic** [comment, marked as answer]
Hi, this is a known XCode 14 issue, which has been fixed on Flutter 3.3.3. Also, look at the detailed solution [here](https://docs.codemagic.io/troubleshooting/common-ios-issues/#xcode-14-known-issues).

**Comment by nmtx-2022** [comment]
@K-Codemagic 

Thanks, so again, what should I do with this error?

![图片](https://user-images.githubusercontent.com/109792150/193219187-e6570fd5-6bae-4460-9df2-bca31d881a5a.png)

**Reply by nmtx-2022** [reply]
I found this: https://docs.codemagic.io/knowledge-codemagic/build-versioning/#app-store-or-testflight-latest-build-number

**Comment by nmtx-2022** [comment]
![图片](https://user-images.githubusercontent.com/109792150/193243563-2f2de522-49a1-4c43-a423-16cb32733a73.png)
